app/views/corporate/_content_items.html.erb

```diff
@@ -1,15 +1,9 @@
 <div class="l-inpage-nav">
```

Comment:
Yeah. It's not ideal but it's due to the way things have already been implemented. One is for rendering corporate category items and the other is for `/en/corporate` items.